cfme/web_ui/__init__.py

```diff
@@ -3399,7 +3399,8 @@ def get_cell(self, row_text, col_index):
             Selenium element of the cell.
         """
         self.expand_all_sections()
-        cell_loc = ".//div/div[1][contains(., '{}')]/../div[{}]".format(row_text, col_index + 2)
+        cell_loc = ".//div/div[1][contains(., '{}')]//tbody/tr[{}]/td".format(
```

Comment:
This xpath isn't matching the driftgrid in 5.7.

suggest trying something more like the following, with a bit appended for the value cell instead of the row_text cell.
`'.//div[@id="compare-grid"]/table//th[contains(normalize-space(.), "{}")]'.format(row_text)`